gui/public/index.html

```diff
@@ -12,6 +12,11 @@
     {%- if version -%}
     window.taipyVersion="{{version}}";
     {% endif %}</script>
+    <style>
```

Comment:
interesting, I have often see the style="vars " (by Microsoft and others ...)

Reply:
It's not uncommon, indeed, but it makes overriding CSS custom properties a little less easy and intuitive.
The good practice in CSS for them is to assign base variables to the :root selector. Problem is this selector has a lower priority than tag selectors like `<html>` so in order to override them, you have to target the html selector, which is counter-intuitive, especially for beginers. 
I'd say, let's set the lowest CSS priority on our stylekit variables so whatever the selector used by the user to reassign them, he/she will see his/her own taken into account instead.